tests/integration/test_dockertar_sha256_helper.sh

```diff
@@ -5,16 +5,16 @@ IFS=$'\n\t'
 DOCKERTAR_SHA256_HELPER=${ATOMIC_LIBEXEC}/dockertar-sha256-helper
 
 T1="${DOCKERTAR_SHA256_HELPER} <(seq 2000)"
-EXPECTED_T1="cc27c088933fbaf64a2374cd1ff39f38df03badac18861f2c3f16e3d78be8f93"
+EXPECTED_T1="0a611a63a42de01613a6d6eb296e469c1f5c3229b36df014e6434e643e2c827e"
 
 T2="${DOCKERTAR_SHA256_HELPER} <(head -c 4096 /dev/zero)"
-EXPECTED_T2="d84f7b85b256694bcb87b6c01777871a2e928fee54b4013e87d04ec4ff844053"
+EXPECTED_T2="afc5adbe422839336d3a77a5a0267ad740461f4f4db9af9eaa2520107a838f1e"
 
 T3="${DOCKERTAR_SHA256_HELPER} <(head -c 4096 /dev/zero | tr '\0' a)"
-EXPECTED_T3="9d974f75dec805bd50bcf0abb82c7e785c3822a3620987b42253ffe78e703639"
+EXPECTED_T3="9e4cd3ee84c4916d40f0067b9aa8c70bfce7067510bdd8346c0d62cf30e568e5"
 
 T4="${DOCKERTAR_SHA256_HELPER} <(head -c 500 /dev/zero | tr '\0' a)"
-EXPECTED_T4="1d83d97035d26a51b6d85ad57d2894afdd74121752b296be874512fd9d85a370"
+EXPECTED_T4="f5dd206f74a9f913b29d495f7d3c2917d221131a7d261ec4293b48df67cdb6bf"
```

Comment:
Interesting; why did the checksums change? Is that something we should be concerned about? (I.e. doesn't that mean all previously calculated checksums are now invalid?).